### DISCUSSION TITLE
正しく集計されていることを確認するテスト

### DISCUSSION THREAD

**Discussion by chikowaka** [ushmz/subsuke#28] [Product backlog]
誰が　　　：
何をしたい：

テスト要件
前提条件　：
手順　　　：
入力値　　：
期待する値：

**Comment by ushmz** [comment]
`price_carousel.dart` にかかれている．

https://github.com/ushmz/subsuke/blob/c62d7496f52b80c30ce9e784960da53b20052095/lib/ui/components/templates/price_carousel.dart#L17-L57

Widget にかかれているので，ロジック用のファイルに分離したほうがいいかもしれない．
